src/PointwiseFunctions/GeneralRelativity/Christoffel.cpp

```diff
@@ -36,15 +36,52 @@ tnsr::abb<DataType, SpatialDim, Frame, Index> christoffel_first_kind(
   christoffel_first_kind(make_not_null(&christoffel), d_metric);
   return christoffel;
 }
+
+template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
+void christoffel_second_kind(
+    const gsl::not_null<tnsr::Abb<DataType, SpatialDim, Frame, Index>*>
+        christoffel,
+    const tnsr::abb<DataType, SpatialDim, Frame, Index>& d_metric,
+    const tnsr::AA<DataType, SpatialDim, Frame, Index>&
+        inverse_metric) noexcept {
```

Comment:
Does this assume the size of christoffel matches the size of d_metric? Why not do something like what you do on lines 19-23 of the GeneralizedHarmonic version (`if UNLIKELY(sizes_not_equal) {resize}`)?

Reply:
Added `if UNLIKELY(sizes_not_equal) {resize}` here too.